src/blenderbim/blenderbim/bim/module/patch/operator.py

```diff
@@ -39,22 +41,55 @@ class ExecuteIfcPatch(bpy.types.Operator):
     bl_idname = "bim.execute_ifc_patch"
     bl_label = "Execute IFCPatch"
     file_format: bpy.props.StringProperty()
+    use_json_for_args: bpy.props.BoolProperty()
 
     @classmethod
     def poll(cls, context):
         input_file = context.scene.BIMPatchProperties.ifc_patch_input
         return os.path.isfile(input_file) and "ifc" in os.path.splitext(input_file)[1]
 
     def execute(self, context):
-        import ifcpatch
```

Comment:
Lazy import because IfcPatch may pull in nasty dependencies which the user may not have.

Reply:
OK, we only want to load it if the user is going to actually use the functionality, right ?

Reply:
I think so, yes. This is the same for all of the auxiliary utilities (IfcClash, IfcCSV, etc)